src/hdmint/tracker.cpp

```diff
@@ -475,7 +475,7 @@ bool CHDMintTracker::UpdateMetaStatus(const std::set<uint256>& setMempool, CMint
 }
 
 void CHDMintTracker::UpdateFromBlock(const std::list<std::pair<uint256, MintPoolEntry>>& mintPoolEntries, const std::vector<CMintMeta>& updatedMeta){
-    if(mintPoolEntries.size()>0)
+    if(mintPoolEntries.size()>0 && zwalletMain)
```

Comment:
No need to check here since tracker should not get invoked any methods if HD mints is not activated.